test/sanity/balances.ts

```diff
@@ -94,20 +81,21 @@ describe('Balance Tests', () => {
 
   it('Check operators earnings in three blocks, one after the other', async () => {
     await utils.progressBlocks(1);
-    expect(await ssvViews.getOperatorEarnings(1)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 2);
-    expect(await ssvViews.getOperatorEarnings(2)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 2);
-    expect(await ssvViews.getOperatorEarnings(3)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 2);
-    expect(await ssvViews.getOperatorEarnings(4)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 2);
+
+    expect(await ssvViews.getOperatorEarnings(1)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 3);
+    expect(await ssvViews.getOperatorEarnings(2)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 3);
+    expect(await ssvViews.getOperatorEarnings(3)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 3);
+    expect(await ssvViews.getOperatorEarnings(4)).to.equal(helpers.CONFIG.minimalOperatorFee * 2 + helpers.CONFIG.minimalOperatorFee * 3);
```

Comment:
Why did all these change from 2 to 3?
"helpers.CONFIG.minimalOperatorFee * 3"

I see its in a lot of places that these values changed, but why would balances be changed or affected from this PR?

Reply:
thats because the extra blocks (time passed) that this kind of extra transactions created in contracts helper: [link](https://github.com/bloxapp/ssv-network/blob/ab7d3220decc8a1e37c64bbc517600855d5c1725/test/helpers/contract-helpers.ts#L123)